include/znc/Message.h

```diff
@@ -39,6 +39,25 @@ class CChan;
 class CClient;
 class CIRCNetwork;
 
+/**
+ * Here is a small explanation of how messages on IRC work, and how you can use
+ * this class to get useful information from the parsed message.
+ *
+ * @time=some-timestamp :server.network.net 366 something #channel :End of /NAMES list.
+ * tags                                     0   1         2        3
+ *
+ * `tags` is the IRCv3 tags associated with the message, obtained with GetTag(s).
```

Comment:
`GetTag("time")`, but time is probably not the best example, because there is also `GetTime()`, which works even if the message didn't have a `time` tag.